content/applications/inventory_and_mrp/purchase/manage_deals/calls_for_tenders.rst

```diff
@@ -1,267 +1,285 @@
 
 .. tip::
-   If a large number of purchase orders are being processed and the previous
-   :abbr:`POs (Purchase Orders)` can't be located, try clicking :menuselection:`Group By -->
-   Vendor` under the search bar at the top of the pop-up window to group by the vendors selected on
-   the previous orders.
+   If a large number of |POs| are being processed and the previous |POs| can't be located, click
+   :menuselection:`Group By --> Vendor` (under the search bar, at the top of the pop-up window) to
+   group the vendors selected on the previous orders.
 
 Compare product lines
 =====================
 
-When there are multiple :abbr:`RFQs (Requests for Quotation)` linked as alternatives, they can be
-compared side-by-side in order to determine which vendors offer the best deals on which products.
-To compare each quotation, go to the :menuselection:`Purchase` app, and select one of the
-quotations created previously.
+Alternative |RfQs| can be compared side-by-side in order to determine which vendors offer the best
+deals on the products included in the orders.
 
-Then, click the :guilabel:`Alternatives` tab to see all the linked
-:abbr:`RFQs (Requests for Quotation)`. Next, under the :guilabel:`Create Alternative` tab, click
-:guilabel:`Compare Product Lines`. This navigates to a Compare Order Lines page.
+To compare alternative |RfQs|, navigate to the :menuselection:`Purchase` app, and select one of the
+previously-created |RfQs|.
 
-.. image:: calls_for_tenders/calls-for-tenders-compare-product-lines.png
+Click the :guilabel:`Alternatives` tab to see all linked |RfQs|. Then, under the :guilabel:`Create
+Alternative` tab, click :guilabel:`Compare Product Lines`. This navigates to the :guilabel:`Compare
+Order Lines` page.
+
+.. image:: calls_for_tenders/calls-for-tenders-compare-products.png
    :align: center
    :alt: Compare Product Lines page for alternative RFQs.
 
-The Compare Order Lines page, by default, groups by :guilabel:`Product`. Each product included in
-any of the :abbr:`RFQs (Requests for Quotation)` is displayed in its own drop-down, along with all
-of the :abbr:`PO (Purchase Order)` numbers in the :guilabel:`Reference` column.
+The :guilabel:`Compare Order Lines` page, by default, groups by :guilabel:`Product`. Each product
+included in any of the |RfQs| is displayed in its own drop-down, and lists all of the |PO| numbers
+in the :guilabel:`Reference` column.
 
 Additional columns on this page include the :guilabel:`Vendor` from which products were ordered,
-the :guilabel:`Status` of the quotation (i.e., RFQ, RFQ Sent); the :guilabel:`Quantity` of products
-ordered from each vendor; the :guilabel:`Unit Price` per product and :guilabel:`Total` price of the
-order, and more.
+the :guilabel:`Status` of the quotation (e.g., RFQ, RFQ Sent), the :guilabel:`Quantity` of products
+ordered from each vendor, the :guilabel:`Unit Price` per product, and the :guilabel:`Total` price of
+the order.
```

Comment:
I'm also seeing columns for: OTD, Description, Expected Arrival, Unit of Measure, and Currency. Are these *not* by default? Either way, it would be wise to double-check to see which columns *do* appear by default, just to make sure everything about this doc is accurate as possible. 👍

Reply:
Added in each of these columns. Suggestion implemented ✅